Pokedex/Pokedex/Models/Character.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  Character.swift
+//  Pokedex
+//
+//  Created by morse on 5/10/19.
+//  Copyright © 2019 morse. All rights reserved.
+//
+
+import Foundation
+
+//struct Character {
+//    let abilities: [String]
+//    let id: Int
+//    let name: String
+//    let types: [String]
+//    let sprites: String
+//}
+
+struct Character: Codable, Equatable {
+    static func == (lhs: Character, rhs: Character) -> Bool {
+        return lhs.name == rhs.name
+    }
+    
+    let abilities: [AbilityDescription]
+    let id: Int
+    let name: String
+    let types: [TypeDescription]
+    let sprites: Sprite
+}
+
+
+
+struct AbilityDescription: Codable {
+    let ability: Ability
+}
+
+struct Ability: Codable {
+    let name: String
+}
+
+struct TypeDescription: Codable {
+    let type: TypeName
+}
+
+struct TypeName: Codable {
+    let name: String
+}
+
+struct Sprite: Codable {
+    let front_default: String
+}
+
+enum CodingKeys: String, CodingKey {
+    case frontDefault = "front_default"
+}
+
+
+//struct Character: Codable {
```

Comment:
Remove commented code that is no longer needed.

You can always go back in history via git to get old code.